src/app/tests/suites/certification/Test_TC_CC_3_1.yaml

```diff
@@ -206,61 +206,41 @@ tests:
               - name: "OptionsOverride"
                 value: 0
 
-    - label: "Wait 10s"
+    - label: "Wait 5s"
       PICS: CC.S.F00
       cluster: "DelayCommands"
       command: "WaitForMs"
       arguments:
           values:
               - name: "ms"
-                value: 10000
+                value: 5000
 
-    #https://github.com/CHIP-Specifications/chip-test-scripts/issues/416#issuecomment-1228072461
     - label: "Step 3c: TH reads CurrentHue attribute from DUT"
-      PICS: CC.S.F00 && CC.S.A0000 && PICS_SKIP_SAMPLE_APP
-      verification: |
-          ./chip-tool colorcontrol read current-hue 1 1
-          Verify response contains a CurrentHue value  in  TH(chip-tool)  Logs:
-
-          [1649661849.854961][8307:8312] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0300 Attribute 0x0000_0000 DataVersion: 4087274517
-          [1649661849.855019][8307:8312] CHIP:TOO:   current hue: 0 (value can vary)
-          [1649661849.855097][8307:8312] CHIP:EM: Sending Standalone Ack for MessageCounter:13414799 on exchange 39735i
-      cluster: "LogCommands"
-      command: "UserPrompt"
-      arguments:
-          values:
-              - name: "message"
-                value: "Enter 'y' after success"
-              - name: "expectedValue"
-                value: "y"
+      PICS: CC.S.F00 && CC.S.A0000
+      command: "readAttribute"
+      attribute: "CurrentHue"
+      response:
+          constraints:
+              minValue: 25
+              maxValue: 35
 
-    - label: "Wait 10s"
+    - label: "Wait 15s"
```

Comment:
same here. My copy of the spec says "After another 10 seconds, TH reads CurrentHue attribute from DUT."